### PR TITLE
Sg/fix lint errors

### PR DESCRIPTION
**Pull request by StephenGregory**
Changelog:

- use double quotes instead of single quotes (The `.eslintrc` specifies using double quotes)
- update the JSON test objects to be valid JSON.

### CODE REVIEW

File: test/json/array.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
              "year":1937
           },
           {
-             "name":Les Demoiselles d'Avignon",
```
Comment (MiguelCastillo) [inline test/json/array.json]: all these tests are supposed to be broken.  They are a quick way to manually verify if the json linter is reporting errors/warnings correctly.

**Comment by MiguelCastillo** [comment]
cool! The PR looks clean.  The JSON file should stay the unchanged since they are broken on purpose to test the JSON linter. :)

File: test/json/array.json (hunk 1 above)
Comment (StephenGregory) [inline test/json/array.json]: Okay, sure. Maybe they could be named to `invalidArray.json` so that the intent is clear?

**Comment by StephenGregory** [comment]
Ok. I've removed the fixes to the JSON files. Should be good to go :smile:

File: test/json/artists.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
                "year":1889
             },
             {
-               'name':"Starry Night Over the Rhone",
+               "name":"Starry Night Over the Rhone",
```
Comment (MiguelCastillo) [inline test/json/artists.json]: This one also needs to reverted.  Single quotes are part of the test.  I know we can improve on this quite a bit. I am open to PRs :D

Comment (StephenGregory) [inline test/json/artists.json]: Good catch! I'll revert this as well.

**Comment by StephenGregory** [comment]
I have reverted my changes to `test/json/artists.json` @MiguelCastillo . _Should_ be good to merge now.

**Comment by MiguelCastillo** [comment]
@StephenGregory Havent merged it because there are conflicts with Master... I will merge when we resolve those issues.  Thank you so much for your PR

**Comment by StephenGregory** [comment]
Conflicts should now be resolved.

**Pull request merged**